### PR TITLE
fix: show toggle theme toggle btn in safari

### PR DESCRIPTION
**Pull request by amareshsm**
### Description :

The theme toggle button is not visible in safari.  Moved theme toggle logic into react app.

| Before | After |
| ------------- | ------------- |
|  image  |  image |

Similar to #74.

### CODE REVIEW

**Review by nzakas — APPROVED** [review]
LGTM. Would like @snitin315 to verify.

File: src/playground/hooks/updateEffect.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    isInitialMount.current = false;
+                }
+                render.current = "second";
+            } else {
+                isInitialMount.current = false;
+            }
+        } else {
+            return effect();
+        }
+        return () => {};
```
Comment (harish-sethuraman) [inline src/playground/hooks/updateEffect.js:22]:
```suggestion
        return () => {};
```
If there is no cleanup this return is not required

File: src/playground/hooks/updateEffect.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export default function useUpdateEffect(effect, dependencies = []) {
+    const render = useRef("first");
+    const isInitialMount = useRef(true);
+
+    useEffect(() => {
+        if (isInitialMount.current) {
+
+            // To handle react scrict mode.
+            if (!process.env.NODE_ENV || process.env.NODE_ENV === "development") {
```
Comment (harish-sethuraman) [inline src/playground/hooks/updateEffect.js:11]: Why do we want to handle the strict mode separately?

**Review by snitin315 — APPROVED** [review]
LGTM

**Pull request merged**